flow/serialize.h

```diff
@@ -276,6 +278,27 @@ inline void load(Archive& ar, std::map<K, V>& value) {
 	ASSERT(ar.protocolVersion().isValid());
 }
 
+template <class Archive, class K, class V>
+inline void save(Archive& ar, const boost::container::flat_map<K, V>& value) {
```

Comment:
It was placed here because other `load` and `save` functions are defined in this file as well. We can move to `fdbclient/VersionVector.h`, but that seems to be inconsistent with existing code.